tsl/test/expected/dist_hypertable-11.out

```diff
@@ -4763,3 +4763,554 @@ SET timescaledb.enable_per_data_node_queries = true;
 SELECT tableoid::regclass, * FROM dist_syscol;
 ERROR:  system columns are not accessible on distributed hypertables with current settings
 \set ON_ERROR_STOP 1
+-- Test SkipScan
+-- Add an index on the value column
+CREATE TABLE hyper (time bigint NOT NULL, device integer);
+CREATE INDEX idx_hyper_dev ON hyper (device);
+SELECT create_distributed_hypertable('hyper', 'time',  chunk_time_interval => 200);
+ create_distributed_hypertable 
+-------------------------------
+ (27,public,hyper,t)
+(1 row)
+
+INSERT INTO hyper SELECT g, g%3 FROM generate_series(0,1000) g;
+-- Ensure that a Unique plan gets chosen on the access node
+SET enable_hashagg TO false;
+SET timescaledb.enable_remote_explain = OFF;
+EXPLAIN (VERBOSE, COSTS OFF, TIMING OFF, SUMMARY OFF)
+SELECT DISTINCT device FROM hyper;
```

Comment:
So, I think these tests might fit better in the `dist_query` test file. That file already has similar DISTINCT test cases and is focused on queries as opposed to this one that focuses on DDL and more general conformance testing.

Reply:
I did think of putting these in `dist_query` but it seemed like a force fit to me. Will try to shift the tests there again.